kubetest/main.go

```diff
@@ -488,6 +490,11 @@ func migrateGcpEnvAndOptions(o *options) error {
 			name:   "--gcp-zone",
 		},
 		{
+			env:    "REGION",
```

Comment:
Seems `migrateOptions` will not set the env if flag does not exist, this should be fine